packages/admin-stories/src/docs/components/Master/Master.stories.mdx

```diff
@@ -0,0 +1,47 @@
+import { Meta, Canvas, Story } from "@storybook/addon-docs/blocks";
+
+<Meta title="Docs/Components/Master" />
+
+# Master
+
+The master components are used to create the root of your application.
+
+```tsx
+<MasterLayout menuComponent={MasterMenu} headerComponent={MasterHeader}>
+    <MainContent>{/* Application content */}</MainContent>
+</MasterLayout>
+```
+
+<Canvas>
+    <Story id="stories-components-master--master" height="500px" />
+</Canvas>
+
+## MasterLayout
+
+`MasterLayout` is used to contain the application's global components, such as the header and the menu.  
+It's children will be the content of the application.
+
+_Note: `MasterLayout` can only be used inside `RouterBrowserRouter`._
```

Comment:
```suggestion
> Note: `MasterLayout` can only be used inside `RouterBrowserRouter`.
```

This looks like this:

> Note: `MasterLayout` can only be used inside `RouterBrowserRouter`.

Reply:
This is't really a correct use-case for a blockquote...

"The blockquote element is used to indicate the quotation of a large section of text from another source." - [Wikipedia](https://en.wikipedia.org/wiki/Blockquote_element)

Reply:
Good point! Up until now, I've never thought about HTML semantics when writing markdown.